spec.html

```diff
@@ -16635,9 +16634,7 @@
         1. Repeat for each |CaseClause| _C_ in _A_,
           1. If _found_ is *false*, then
             1. Let _clauseSelector_ be the result of CaseSelectorEvaluation of _C_.
-            1. If _clauseSelector_ is an abrupt completion, then
-              1. If _clauseSelector_.[[value]] is ~empty~, return Completion{[[type]]: _clauseSelector_.[[type]], [[value]]: *undefined*, [[target]]: _clauseSelector_.[[target]]}.
-              1. Else, return Completion(_clauseSelector_).
+            1. ReturnIfAbrupt(_clauseSelector_).
```

Comment:
Empty completion values are (currently *) not possible here, since the case selector is an expression and expression evaluation always produces an value.

(*) Obviously this needs to be changed when/if do-expressions are added.